Mathlib/Control/Monad/Cont.lean

```diff
@@ -41,6 +41,16 @@ class LawfulMonadCont (m : Type u → Type v) [Monad m] [MonadCont m] : Prop
 
 export LawfulMonadCont (callCC_bind_right callCC_bind_left callCC_dummy)
 
+/-- The continuation monad transformer. Given a return type `r`, a monad `m`, and a type `α`,
+it represents computations that take a continuation function from `α` to `m r` and return an `m r`.
```

Comment:
It seems more general in that we only need a type transformer. A more fleshed out example would be good also.

Reply:
I will simply delete the example, since I don't see anything similar in the analogous Haskell documentation.